apps/vaporgui/VaporWidgets.h

```diff
@@ -51,12 +51,12 @@ class VaporWidget : public QWidget
 //
 // ====================================
 //
-class VSpinBox : public VaporWidget
+class VSpinBoxOldOld : public VaporWidget
```

Comment:
What is with the "Old" classes? Is this a temporary name that is needed for some reason?

Reply:
You're right, those are legacy.  Deleted.